### PR TITLE
Delete shared directories

### PR DESCRIPTION
**Pull request by KKoukiou**
Machines: Delete shared directories
------------------------------------------------

Cockpit Machines can now delete existing filesystem devices.

Screen Shot 2021-08-10 at 15 55 54

### CODE REVIEW

File: test/check-machines-filesystems [hunk 1]
```diff
@@ -116,6 +116,23 @@ class TestMachinesFilesystems(VirtualMachinesCase):
         b.click("#vm-subVmTest1-run")
         b.wait_visible("#vm-subVmTest1-filesystems-add[aria-disabled=true]")
 
+    @skipImage("Older libvirt does not support virtiofs", "rhel-8-5", "centos-8-stream", "debian-stable", "ubuntu-2004", "fedora-33")
```
Comment (marusak) [inline test/check-machines-filesystems:119]: How does then UI behave? Do we have test for it?

**Review by marusak — APPROVED** [review]
Thank you!

**Comment by garrett** [comment]
I just saw this in the temporary release notes and I think deleting `/mnt` is a very, very _bad_ idea. Recursively deleting directories (and files) in the UI is totally dangerous.

If you mean "Remove", then we need to hotfix this. Remove is not delete.

**Comment by garrett** [comment]
Quoting https://www.patternfly.org/v4/ux-writing/terminology/:

> **Add**: (v.) Use to describe adding an existing item to an existing list, group, view, or other container element.
>
> - Opposite: Remove
> - Notes: If the object being added is not readily apparent from the context, consider adding a noun (like “Add user”). If you’re creating a new object, do not use *Add*. See *Create*.
    

> **Create**: (v.) Use to describe creating something new.
> 
> - Opposite: Delete
> - Notes: If the object being created is not readily apparent from the context, consider adding a noun (like “Create user”). *New* or *Add* are not recommended for this use case. See *Add* for usage guidelines.
> 
> **Delete**: (v.) Use to describe completely removing an object. Delete is a destructive action.
> 
> - Opposite: Create
> - Notes: *Erase* and *Remove* are not recommended for this use case. See *Remove* for usage guidelines.

(Remove itself is missing on the page. I guess someone removed "remove"?)

**Pull request merged**